kalcy/src/scanner.cpp

```diff
@@ -24,8 +24,8 @@ auto Scanner::next() -> Token {
 	if (at_end()) { return {}; }
 
 	auto out = Token{};
-	if (match_number(out)) { return out; }
 	if (match_single(out)) { return out; }
+	if (match_number(out)) { return out; }
```

Comment:
Why are these changes here? Did you not rebase / merge with `origin/main`?

Reply:
I have, and even the `-` bug is fixed.